zebra-state/src/arbitrary.rs

```diff
@@ -26,3 +32,104 @@ impl Prepare for Arc<Block> {
         }
     }
 }
+
+impl PreparedBlock {
+    /// Returns a [`ContextuallyValidBlock`] created from this block,
+    /// with fake zero-valued spent UTXOs.
+    ///
+    /// Only for use in tests.
+    #[cfg(any(test, feature = "proptest-impl"))]
```

Comment:
These are redundant after I moved this code into the arbitrary module.
```suggestion
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2652/commits/a874be4419ffe66430d5d1cb845f24fe75588e85